public/locales/en/translations.json

```diff
@@ -0,0 +1,51 @@
+{
+	"root.title": "aux.computer",
```

Comment:
I'm not 100% certain this should be included in the translation file. It's the main domain of the website (for now), and shouldn't be changed in different language versions.